vue-element-admin/build.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/env bash
+
+set -e # exit when any command fails
+
+cd vue-element-admin
+npm install
+PUBLIC_URL=/vue-element-admin npm run build:prod
```

Comment:
Great! The deployment worked. Now the problem is that `PUBLIC_URL` is not recognized by their build, so we should find a way to customize that URL — if it exists

Reply:
Yes, still trying out something by modify the vue.config.js

Reply:
It's now working though you have to append index.html at the end just like in the link below 
[Vue-element-admin](https://playground-b1751et14-pixiebrix.vercel.app/vue-element-admin/index.html)
index file not being read automatically as it should be

Reply:
Hi @EmmanuelObua, I think you have to add a rewrite rule to vercel.json like we did for the react project: http://github.com/pixiebrix/playground/blob/3c5495d846f1d2852e82ac04a090c22ff312acc7/vercel.json#L5-L5